tests/test_backwards_compatibility.py

```diff
@@ -1,9 +1,12 @@
 import os
 
 import pytest
+from conftest import DATA, LEGACY
```

Comment:
Isn't it generally discouraged to import from `conftest` and instead use fixtures?